src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -15,6 +15,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <configurator>
+  <bitmap name="NameSupportBitmap" type="BITMAP8">
```

Comment:
This is not in the spec.  Please do not introduce divergences wrt the spec...

Reply:
Spec PR opened here: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7644

Reply:
Well, that PR does not match this naming, so....

Reply:
Origially it didn't but now it does for [NameSupportBitmap](https://github.com/lpbeliveau-silabs/connectedhomeip-spec/blob/feature/scenes_name_support_bitmap/src/app_clusters/Scenes.adoc#ref_SceneNameSupportBitmap)